frontend/packages/component-library/src/cms/section/__tests__/Section.test.tsx

```diff
@@ -0,0 +1,40 @@
+import {render, screen} from '@testing-library/react';
+import Section, {SectionProps} from '../Section';
+
+describe('Section Component', () => {
+  const renderComponent = (props: Partial<SectionProps> = {}) => {
+    return render(
+      <Section {...props}>
+        <div>This is content.</div>
+      </Section>,
+    );
+  };
+
+  it('renders children content', () => {
+    renderComponent();
+
+    // check if children content is in the document
+    expect(screen.getByText('This is content.')).toBeInTheDocument();
+  });
+
+  it('changes background color based on props', () => {
+    const {rerender} = renderComponent({backgroundColor: 'secondary'});
```

Comment:
super nit, non blocking: since we moved `sectionBackgroundColors` to a constant - we can use this constant always instead of passing strings.
Might be worth using `sectionBackgroundColors` in tests and stories to give consumers better understanding that we have `sectionBackgroundColors` and intended approach is to use them.

That being said, it would concern me more if it was general dsco component, but since it's just cms for now - it's definitely non blocking

Reply:
Oh yes, I love this and am learning so much! 🙏  Updated here: https://github.com/code-dot-org/code-dot-org/pull/63706/commits/dd52d481bf12d4802c64d6db7f2aefb98cbf1ac6